octoprint_mrbeam/static/js/mrbeam.js

```diff
@@ -443,6 +445,22 @@ $(function () {
         };
 
         self._handle_session_expired = function (triggerUrl) {
+            if (self.isCurtainOpened > 0) {
+                self.error401Count++;
+                if (self.error401Count === 1) {
+                    setTimeout(() => {
+                        let error401Count = self.error401Count;
+                        let payload = {
+                            error401Count: error401Count,
+                        };
+                        self.analytics.send_fontend_event(
+                            "expired_session",
+                            payload
+                        );
+                        self.error401Count = 0;
+                    }, 10000);
```

Comment:
Please set the timeout to 2000.
```suggestion
                    }, 2000);
```